### PR TITLE
Mobile Responsive CSS and other css focused tweaks

### PR DESCRIPTION
**Pull request by dkpat12**

### CODE REVIEW

**Comment by KevinPayravi** [comment]
@dkpat12 This is great stuff, thank you.

https://github.com/Nintendo-Independent-Wiki-Alliance/niwa-website/pull/25 has been merged, so this PR is ready to have conflicts resolved.

**Comment by dkpat12** [comment]
Ready to go now!

File: _footer.php [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-<div id="footer"><div style="float: left;"><a href="/legal.php">Legal</a></div>
-<a href="/">Nintendo Independent Wiki Alliance</a> &copy; 2010-<?php echo(date('Y')); ?>
+<div id="footer">
+    <div style="float: left;"><a href="/legal.php">Legal</a></div>
+    <a href="http://niwanetwork.org">Nintendo Independent Wiki Alliance</a> &copy; 2010-<?php echo (date('Y')); ?>
 </div>
 
 </div>
 
-<script src="https://apis.google.com/js/plusone.js"></script>
```
Comment (dkpat12) [inline _footer.php:9]: I just went ahead and killed this while fixing the conflicts

**Review by KevinPayravi — APPROVED** [review]
LGTM

**Pull request merged**